tianshou/policy/base.py

```diff
@@ -259,6 +259,12 @@ def map_action_inverse(
                     act = (np.log(1.0 + act) - np.log(1.0 - act)) / 2.0  # type: ignore
         return act
 
+    def process_buffer(
```

Comment:
Pls add documentation on when and how often this is supposed to be called. Also, why should it be possible to accept and return None?

Reply:
Because it is used in BaseTrainer init method where base trainer accepts None in buffer argument, but I guess I can add an if statement to rule out None.